config/webpack.config.js

```diff
@@ -11,10 +11,21 @@ var LodashModuleReplacementPlugin = require('lodash-webpack-plugin');
 var pluginUtils = require('../script/plugin_webpack_directories');
 var vendorEntry = require('./webpack.vendor');
 var SimpleNamedModulesPlugin = require('../webpack/simple_named_modules');
+var { execSync } = require('child_process');
+
+ const getHostname = () => {
```

Comment:
I would suggest to call the argument bindhost instead of hostname (to alignt to rails, webpack etc).
also, in order to get the hostname in node, please use something like
```js
var os = require('os');

const devServerHostname = process.env.HOSTNAME || os.hostname();
```